pkcs11/Cargo.toml

```diff
@@ -13,3 +13,4 @@ bindgen = "0.71"
 
 [features]
 pkcs11_3_2 = []
+default = ["pkcs11_3_2"]
```

Comment:
I think we should remove it here as it really does not do anything anymore. Or is it something for "backward compatibility"?